swagger/swagger-generator/generator-core/src/main/java/org/apache/servicecomb/swagger/generator/core/processor/annotation/ApiProcessor.java

```diff
@@ -17,18 +17,69 @@
 
 package org.apache.servicecomb.swagger.generator.core.processor.annotation;
 
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 import org.apache.servicecomb.swagger.generator.core.ClassAnnotationProcessor;
 import org.apache.servicecomb.swagger.generator.core.SwaggerGenerator;
 import org.springframework.util.StringUtils;
 
 import io.swagger.annotations.Api;
+import io.swagger.models.Swagger;
 
 public class ApiProcessor implements ClassAnnotationProcessor {
   @Override
   public void process(Object annotation, SwaggerGenerator swaggerGenerator) {
     Api api = (Api) annotation;
 
     setTags(api, swaggerGenerator);
+    // except for @Api, @RequestMapping can also set consumes and produces
+    // @RequestMapping takes HIGHER priority than @Api for legacy reason
+    processConsumes(api, swaggerGenerator.getSwagger());
```

Comment:
i can not find the priority control

Reply:
Both `Api` and `RequestMapping` can set consumes and produces but their processors' behavior is different. `ApiProcessor` only set consumes/produces when there is no config in Swagger, but `RequestMappingClassAnnotationProcessor` will set consumes/produces no matter there is config in Swagger.
In other words, `ApiProcessor` cannot over write the config but `RequestMappingClassAnnotationProcessor` can. So `RequestMapping` takes higher priority.

Reply:
ok